### ISSUE TITLE
c_date gets decreased by 1 day when doing "PkgTool.Core pkg_build"

### ISSUE THREAD

**Issue by hippie68** [maxton/LibOrbisPkg#73]
The "c_date" variable from param.sfo and inside Project.gp4 gets decreased by 1 day when you build a PKG and then look at it with sfo_listentries or extract it with pkg_makegp4. Probably a bug? If you do extract it with pkg_makegp4, then the newly created Project.gp4's c_date is 1 day behind.

Plus after this, the key PUBTOOLINFO, which formerly included c_date and other info, is totally missing from the extracted param.sfo. Is that intended?

**Comment by maxton** [comment]
Yeah the dates are probably screwy because of time zone conversions. Fortunately it doesn't affect anything. And the pubtoolinfo is removed because it gets set at PKG build time

**Comment by hippie68** [comment]
Right, time zones... I set my times to 00:00:00, so that's probably why.
